solutions/instances/variables.tf

```diff
@@ -310,7 +317,7 @@ variable "management_endpoint_type_for_bucket" {
 variable "existing_kms_instance_crn" {
   type        = string
   default     = null
-  description = "The CRN of the key management service (KMS) that is used for the Cloud Object Storage bucket root key. If you are not using an existing KMS root key, you must specify this CRN. If the existing Cloud Object Storage bucket details are passed as an input, this value is not required."
+  description = "The CRN of the KMS instance (Hyper Protect Crypto Services or Key Protect). Applies only if `existing_kms_instance_crn` or `existing_cos_kms_key_crn` is not specified. If the KMS instance is in a different account, provide a value for `ibmcloud_kms_api_key`."
```

Comment:
The description seems to reference itself `existing_kms_instance_crn`. Did you mean to reference the existing bucket variables?

Reply:
That's my fault. I added that in my review. At least I knew I might be wrong... (`Make sure variable names here are correct.`)